invenio_records_permissions/generators.py

```diff
@@ -103,26 +103,54 @@ def query_filter(self, record=None, **kwargs):
 
 
 class AnyUserIfPublic(Generator):
-    """Allows any user if record is public.
+    """Allows any user if record is open access.
```

Comment:
Mayor: TBD, But I would prefer not changing the existing generators until the exact issues have been clarified. I would create a different one, namely `AnyUserIfOpen`.

Reply:
Yes, let's wait to see what's the verdict on inveniosoftware/rfcs/pull/12 . I will wait until this is resolved to address this.